src/tools/namespace_tool.cpp

```diff
@@ -40,6 +40,8 @@ DEFINE_string(throttleType, "", "throttle type");
 DEFINE_uint64(limit, 0, "throttle limit");
 DEFINE_int64(burst, -1, "throttle burst");
 DEFINE_int64(burstLength, -1, "throttle burst length");
+DEFINE_uint64(stripeUnit, 32 * 1024, "stripe unit size");
+DEFINE_uint64(stripeCount, 32, "strip count");
```

Comment:
for compatibility, I think we shouldn't create stripe volume by default, so these default values should be 0.

Reply:
fix

Reply:
> for compatibility, I think we shouldn't create stripe volume by default, so these default values should be 0.

fix